src/margin.rs

```diff
@@ -0,0 +1,30 @@
+use crate::TableOption;
+use papergrid::{Grid, Indent};
+
+//todo add description
```

Comment:
It would be cool to make it `todo` happen :)

Maybe a doc example would be also good.